docs/src/index.rst

```diff
@@ -1,18 +1,22 @@
 iris-esmf-regrid
 ================
 
-A collection of structured and unstructured ESMF_ regridding schemes for Iris_.
+Iris-esmf-regrid is a package which aims to bridge the gap between Iris_ data
+handling and ESMF_ regridding. This is done largely by way of providing
+Iris_-like regridding schemes.
 
-This rendered documentation is so far limited to iris-esmf-regrid's API. Some
-basic examples and change logs can be found on GitHub_.
+This documentation is a work in progress and some pages will be incomplete.
 
 :doc:`Click to view the API<_api_generated/modules>`
```

Comment:
Is this still appropriate now there are multiple documentation sections? There is also a link in the top bar.

![image](https://github.com/SciTools-incubator/iris-esmf-regrid/assets/40734014/898e3e8f-038d-48f5-acff-339098c23e65)

Reply:
yeah, I'm still thinking of how to arrange links within the index page, I'm thinking I might want a little bit of prose with particalar links in them.